backend/src/models/resources.js

```diff
@@ -9,6 +9,18 @@ const findAll = () => {
   return db(table);
 };
 
+// Get specific one by id.
+const findById = (id) => {
+  return db(table).where("id", id);
+};
+
```

Comment:
Looking at the data, we should be doing lookups for supplier_id, subcontractor_id and user_id

Reply:
yes.. i'm going to create tickets for the index, and for the form

Reply:
It should be done now!!!!!!!!!!! JK :)